1/src/_backend/index.js

```diff
@@ -1,12 +1,26 @@
 const { ApolloServer } = require('apollo-server');
-const typeDefs = require('./schema');
-const resolvers = require('./resolvers');
+const schema= require('./schema');
+//const resolvers = require('./resolvers');
 
 
+const neo4j = require('neo4j-driver').v1;
```

Comment:
If you setup babel like here: https://github.com/m-oehme/Systems-Development-and-Frameworks/pull/6

You could import it like this:
```js
const { v1 as neo4j } from 'neo4j-driver'
```